packages/example/components/chains/conflux/params.ts

```
@@ -165,13 +171,37 @@ export default {
+        'description': '发送 CRC20 代币 Moon',
+        'value': JSON.stringify({
+          from: from,
+          to: "cfx:achcuvuasx3t8zcumtwuf35y51sksewvca0h0hj71a",
+          data: "0xa9059cbb" +
+                address.decodeCfxAddress(to).hexAddress.toString('hex').slice(2).padStart(64, '0') +
+                "00000000000000000000000000000000000000000000000000005af3107a4000",
+        }, null, 2)
+      },
+      {
+        'id': 'sendTransaction-crc20-usdt',
+        'name': 'CRC20 Token Transfer(USDT)',
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)